airbyte-integrations/connectors/source-stock-ticker-api-cdk/unit_tests/test_incremental_streams.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+
+import pytest
+from airbyte_cdk.models import SyncMode
+from source_stock_ticker_api_cdk.source import StockPrices
+
+
+@pytest.fixture
+def stream():
```

Comment:
test_streams.py has similar fixture, so better use conftest file to follow DRY principal.

Reply:
Fixed